src/Alerts/alert.style.js

```diff
@@ -0,0 +1,50 @@
+import styled, { css } from "styled-components";
+import { SUCCESS, WARNING } from "./alertTypes";
+
+export const AlertStyles = styled.div`
+  ${({ normal }) =>
+    normal &&
+    css`
+      padding: 16px;
+      border-radius: 8px;
```

Comment:
correct me if I am wrong, but i think since this is always gonna be a style of this component it can be extracted above. between line 4 and 5.

Reply:
yes, good catch!